src/app/common/subscriptions-component.ts

```diff
@@ -2,9 +2,10 @@
  * Class that will unsubscribe from all observables in a `subscriptions` array
  */
 
-import { OnDestroy } from '@angular/core';
+import { OnDestroy, Directive } from '@angular/core';
 import { Subscription } from 'rxjs';
 
+@Directive()
```

Comment:
There was a note in the upgrade documentation that this needed an @Directive iirc. However, I can try making it abstract.

Reply:
Now that I think about it, I guess it doesn't really matter if we mark it abstract or not since we never construct it directly anyways. Does it fail to compile if there's no decorator?